secscanner2junit/sast.py

```diff
@@ -61,6 +61,12 @@ def parse_findings(self, finding, time):
             output += "end line: {end_line}\n".format(end_line=end_line)
         except KeyError:
             pass
+        
+        try:
+            output += "identifiers.name: {identifiers_name}\n".format(identifiers_name=['identifiers'][0]['name'])
+            output += "identifiers.value: {identifiers_value}\n".format(identifiers_value=['identifiers'][0]['value'])
```

Comment:
- [ ] `identifiers_name=['identifiers'][0]['value']` should be `identifiers_name=finding['identifiers'][0]['value']`

Reply:
fixed, sorry for stupid error ...